.github/workflows/continuous-integration.yml

```diff
@@ -107,16 +100,16 @@ jobs:
       - name: "Install dependencies with Composer"
         uses: "ramsey/composer-install@v1"
 
-      - name: "Run PHPUnit (with coverage)"
-        run: "php vendor/bin/simple-phpunit -v --coverage-clover coverage/clover.xml"
+      - name: "Install dev tools"
+        run: "make install-dev-tools"
+
+      - name: "Run PHPUnit"
+        run: "php dev-tools/vendor/bin/phpunit -v --coverage-clover coverage/clover.xml"
```

Comment:
Should it be better to use `make run-phpunit ....` instead to hide complexity ?

Reply:
Done using optional `$(ARGS)` parameter, see https://github.com/smalot/pdfparser/pull/383/commits/a65d972f0a30ed7fe5e95fdf7e307893099c3fef

If you have a better idea how to do that, please tell me. I am not a Makefile geek.